### PR TITLE
Add storyboard for SMHI Petter

### PR DESCRIPTION
**Pull request by geek-yang**
Create storyboard based on the slides from Petter (SMHI).

Note that in this PR, small corrections to the content of the storyboard from Hlyke (KNMI) based on his feedback is also included (see his email). The changes are too small to fit for a individual PR, therefore they are included here.

### CODE REVIEW

File: content/cp_benefits/1_intro.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+image: cp_benefits/modelling.png
+headline: Convection-permitting modeling in Fenno-Scandinavia
+---
+## Benefits and added value of convection-permitting climate modeling over Fenno-Scandinavia
+
+Snow, ice, the long coastline, and the Scandes mountains, they all make the Fenno-Scandinavia a complex region. The box plots show observed distribution of daily mean T2m (top panel) and Pr (bottom) for DJF (left half) and JJA (right half) seasons.
```
Comment (Peter9192) [inline content/cp_benefits/1_intro.md]: Maybe write out the abbreviations (T2m, Pr, DJF, JJA, ...)

Comment (Peter9192) [reply content/cp_benefits/1_intro.md]: Also in other panels

**Review by Peter9192 — APPROVED** [review]
Overall a very nice storyboard with great quality figures! A comment on the content would be that on some panels, only the figures are explained, but the interpretation is missing.

File: content/cp_benefits/1_intro.md (hunk 1 above)
Comment (geek-yang) [reply content/cp_benefits/1_intro.md]: Thanks for the comment! I will spell out all these abbreviations.

**Comment by Peter9192** [comment]
Did you also report back to the author?

**Pull request merged**